jsx/Tabs.js

```diff
@@ -141,6 +141,123 @@ Tabs.defaultProps = {
   updateURL: false
 };
 
+/**
+ * Allows to dynamically render vertical tabs corresponding to tab panes.
+ */
+
+class VerticalTabs extends React.Component {
+  constructor(props) {
+    super(props);
+
+    const hash = window.location.hash;
+    let activeTab = "";
+
+    /**
+     * Determine the initial active tab in this order
+     * 1. Try to infer from the URL, otherwise
+     * 2. Try to infer from the defaultTab prop, otherwise
+     * 3. Set to be the first tab of the list
+     */
+    if (this.props.updateURL && hash) {
+      activeTab = hash.substr(1);
+    } else if (this.props.defaultTab) {
+      activeTab = this.props.defaultTab;
+    } else if (this.props.tabs.length > 0) {
+      activeTab = this.props.tabs[0].id;
+    }
```

Comment:
else throw an exception?

Reply:
copy of line 51 - regular tabs

@driusan can this PR add it to both ?

Reply:
sure